src/azul/service/manifest_service.py

```diff
@@ -113,6 +113,15 @@ class ManifestFormat(Enum):
 
 @attr.s(auto_attribs=True, kw_only=True, frozen=True)
 class Manifest:
+    """
+    A collection of details for a prepared manifest.
+
+    Attributes:
+        location    A URL to the manifest file.
```

Comment:
```suggestion
        location    The URL of the manifest file.
```

L stands for location